org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/EclipseSbtBuildManager.scala

```diff
@@ -77,11 +77,31 @@ class EclipseSbtBuildManager(val project: IScalaProject, settings0: Settings) ex
 
   override def build(addedOrUpdated: Set[IFile], removed: Set[IFile], pm: SubMonitor) {
     sbtReporter.reset()
+    buildInternally(addedOrUpdated, removed, pm)
+  }
+
+  override def resetReporter(): Unit = sbtReporter.reset()
+
+  override def buildBulk(addedOrUpdated: Set[IFile], removed: Set[IFile], pm: SubMonitor, bulkFilter: Set[IFile] => Set[IFile] ) {
+    buildInternally(addedOrUpdated, removed, pm, bulkFilter)
+  }
+
+  override def clean(implicit monitor: IProgressMonitor) {
+    cachePath.refreshLocal(IResource.DEPTH_ZERO, null)
+    cachePath.delete(true, false, monitor)
+    clearCached()
+  }
+
+  override def invalidateAfterLoad: Boolean = true
+
+  def findInstallation(project: IScalaProject): IScalaInstallation = project.effectiveScalaInstallation()
```

Comment:
Sorry, I see you just moved this method higher in the file. So feel free to disregard my preaching, but I'd still find it useful to remove it. Probably it used to do something more useful before :)